.codecov.yml

```diff
@@ -8,3 +8,4 @@ coverage:
 
 ignore:
   - "**/*generated*.go"
+  - "**/ssar.go"
```

Comment:
why ignore this file?

Reply:
The code in this file mostly consists of construction functions. It not ignored then I need to improve coverage somewhere else.

Reply:
The code here is just move from other files? I think the existing ut has covered them already, why this movement will impact coverage?

Reply:
It moves from one package to another, so when caculate the average code coverage score, it's 80% to (0%+80%)/2 and cause a drop.

I'm working add u-t in other files, I will improve the score back.